cmd/k8s-operator/proxygroup.go

```diff
@@ -237,7 +246,20 @@ func (r *ProxyGroupReconciler) maybeProvision(ctx context.Context, pg *tsapi.Pro
 	r.ensureAddedToGaugeForProxyGroup(pg)
 	r.mu.Unlock()
 
-	cfgHash, err := r.ensureConfigSecretsCreated(ctx, pg, proxyClass)
+	ports := make(map[string]int32)
+	var targetPort *int32
+	var err error
+	if proxyClass != nil &&
+		proxyClass.Spec.TailnetListenerConfig != nil &&
+		proxyClass.Spec.TailnetListenerConfig.Type ==
+			tsapi.TailnetListenerConfigMode(corev1.ServiceTypeNodePort) {
```

Comment:
You added a const for this shttps://github.com/tailscale/tailscale/pull/16115/files#diff-28d073c51227099fd0bfd92891e7b7207e208351aceeae796cd79374a7e5bc1fR108

Reply:
hmmm, where?